mk/mono.mk

```diff
@@ -1,4 +1,4 @@
-NEEDED_MONO_VERSION := bfcac9bd309a05bcc1820e9ff1cb167708e4d6f4
+NEEDED_MONO_VERSION := df42020fe6f7a0a1b5fa59b481fb603d88d72348
 NEEDED_MONO_BRANCH := 2019-10
```

Comment:
This doesn't need to be bumped too?

Reply:
no, we only care about the hash. Master is atm in the same mono as d16-5 but we have the mono integration branch for the bump to 2019-12.

Bumping branches in mono (rather than a hash) is more dangerous and that is why we have the mono monthly integration branch, we use it to find possible mono issues or issues with our code and then merge it to master.